package-scripts.js

```diff
@@ -185,6 +185,10 @@ module.exports = {
         esm: {
           script: 'MOCHA_TEST=esm nps test.browser.unit',
           description: 'Test mocha ESM support'
+        },
+        amd: {
+          script: 'if grep -q "define.amd" mocha.js; then exit 1; fi',
```

Comment:
this will likely fail in windows.  so maybe better to write an actual test (put in`test/` somewhere) which:

1. triggers `npm start build` to ensure `mocha.js` is present
1. reads `mocha.js` into a string via `fs.readFile()`
1. asserts `"define.amd"` is not present in the string

what do you think?

Reply:
I got what you mean. It makes sense and looks like more maintainable.
I will try to fix it.